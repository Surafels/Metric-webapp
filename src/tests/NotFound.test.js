import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import CompanyNotFound from '../components/CompanyNotfound';

describe('CompanyNotFound', () => {
  it('should render correctly', () => {
    const { asFragment } = render(
      <BrowserRouter>
        <CompanyNotFound />
      </BrowserRouter>,
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
