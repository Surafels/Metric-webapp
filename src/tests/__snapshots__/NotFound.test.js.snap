// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CompanyNotFound should render correctly 1`] = `
<DocumentFragment>
  <div
    class="no-company"
  >
    <h2>
      Page Not Found
    </h2>
    <p
      class="go-page"
    >
      <a
        href="/"
      >
        Got to Home page
      </a>
    </p>
  </div>
</DocumentFragment>
`;
